packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -182,6 +182,13 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
   data?: any;
 }
 
+export interface IButtonMenuProps extends IContextualMenuProps {
+  /**
+   * Optional callback when menu is clicked.
+   */
+  onMenuClick?: (ev?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>, button?: IButtonProps) => void;
```

Comment:
Should this be directly on the button props instead?

Reply:
I debated that. Not 100% sure why I went that way, but I'm good with either approach.